pkg/tracker/runner.go

```diff
@@ -7,39 +7,40 @@ import (
 	"context"
 	"time"
 
-	"github.com/ethereum/go-ethereum/common"
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 	"github.com/pkg/errors"
 	tellorCommon "github.com/tellor-io/telliot/pkg/common"
 	"github.com/tellor-io/telliot/pkg/config"
-	"github.com/tellor-io/telliot/pkg/contracts/tellor"
 	"github.com/tellor-io/telliot/pkg/db"
 	"github.com/tellor-io/telliot/pkg/rpc"
 )
 
 // Runner will execute all configured trackers.
 type Runner struct {
-	client       rpc.ETHClient
 	db           db.DB
+	client       rpc.ETHClient
+	contract     *tellorCommon.Contract
+	account      *tellorCommon.Account
 	readyChannel chan bool
 	logger       log.Logger
+	config       *config.Config
 }
 
 // NewRunner will create a new runner instance.
-func NewRunner(client rpc.ETHClient, db db.DB, logger log.Logger) (*Runner, error) {
-	return &Runner{client: client, db: db, readyChannel: make(chan bool, 1), logger: log.With(logger, "component", "runner")}, nil
+func NewRunner(logger log.Logger, config *config.Config, db db.DB, client rpc.ETHClient, contract *tellorCommon.Contract, account *tellorCommon.Account) (*Runner, error) {
+	return &Runner{config: config, db: db, client: client, contract: contract, account: account, readyChannel: make(chan bool, 1), logger: log.With(logger, "component", "runner")}, nil
 }
 
 // Start will kick off the runner until the given exit channel selects.
 func (r *Runner) Start(ctx context.Context, exitCh chan int) error {
-	cfg := config.GetConfig()
+	cfg := *r.config
```

Comment:
Why do you think that this is needed? Why not just use `r.config` directly?
```suggestion
```

Reply:
I don't know, I thought it wouldn't work.